CHANGELOG.md

```diff
@@ -54,6 +54,9 @@
 
 ## v1.30.0 - TBD
 
+### BigQuery:
+* The BigQuery library is now GA. Currently the v1 Storage API is supported.
```

Comment:
It would be to also indicate any changes users (if any) may need to make. For example, "Any users who were previously  using the experimental library may need to remove the "experimental" part from namespaces, build targets, etc."

Also, this is an exciting announcement. We may want to call it out a bit more. Consider adding a new section like:

```markdown
## New GA Libraries
* *BigQuery Storage* -- The [BigQuery storage library](link to code in repo) is now GA. ...
```

Then you'd have a spot to add the announcement for IAM in the subsequent PR.